.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Continuous Integration
+
+on: pull_request
```

Comment:
After reading the documentation I discovered something new for me: pull_request builds not the tip of the branch, but a (potential) merge commit with the master (branch where PR is targeted). So in reality we need both actions. You can see it in the log here: https://github.com/organicmaps/wikiparser/actions/runs/5126383678/jobs/9220795027#step:2:101

That's likely should be checked/fixed in the Organic Maps repo too.

Reply:
Can you please also add some exceptions to CI, like README, license, .gitignore?

Reply:
> So in reality we need both actions.

If you want to avoid extra CI runs, I think using only `pull_request` is still reasonable.
It would be strange if something passes `pull_request` but fails `push`, and what I want to know is "Will this break things if it's merged", which is what `pull_request` does.